File: September/17th September,2024/Leetcode(884).py
class Solution:
    def uncommonFromSentences(self, s1: str, s2: str) -> List[str]:
        a=s1.split()
        b=s2.split()
        d=a+b
        c=Counter(d)
        # for i in d:
        #     if c.get(i) is not None:
        #         c[i]+=1
        #     else:
        #         c[i]=1
        # print(c.keys())
        ans=[]
        for i in c.keys():
            if c[i]==1:
                ans.append(i)
        return ans

File: September/17th September,2024/Leetcode(884).js
/**
 * @param {string} s1
 * @param {string} s2
 * @return {string[]}
 */
var uncommonFromSentences = function(s1, s2) {
    let a=[...s1.split(" "),...s2.split(' ')]
    let ans=[]
    let b={}
    console.log(a)
    let d=new Map();
    for(let i of a){
        if(i in b) b[i]+=1
        else b[i]=1
    }
    for(let i in b){
        if(b[i]==1) ans.push(i)
    }
    return ans
};